pages/development/local-mode.md

```diff
@@ -10,10 +10,8 @@ Local mode is the new experimental development mode for resin.io. It allows you
 
 In order to use local mode on a device:
 * The device must be running resinOS v2.0 or higher with supervisor v4.0 or higher.
-* The device must be running a [development](https://docs.resin.io/understanding/understanding-devices/2.0.0/#dev-vs-prod-images) variant of the OS.  
-  If you try to use a production variant, you will not be able to use local mode. The production devices have SSH and the Docker socket locked down, both of which are needed for the local mode feature.
-* Local mode must be enabled through the dashboard.  
-  To use local mode on a development device, click on the small *Actions* dropdown at the top right of the device page and select *Enable Local Mode*.
+* The device must be running a [development][development] variant of the OS. If you try to use a production variant, you will not be able to use local mode. The production devices have SSH and the Docker socket locked down, both of which are needed for the local mode feature.
+* Local mode must be enabled through the dashboard. To use local mode on a development device, click on the small *Actions* dropdown at the top right of the device page and select *Enable Local Mode*.
```

Comment:
Could we use bold instead of italics? For me, using italics for those keywords wasn't catching my attention.
(Just a question, not a request.)